website/screens/components/tabs/specs/TabsSpecsPage.tsx

```diff
@@ -0,0 +1,312 @@
+                <Code>selectedIconColor</Code>
+              </td>
+              <td>Icon</td>
+              <td>
+                <Code>color-purple-700</Code>
+              </td>
+              <td>#5f249f</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>unselectedIconColor</Code>
+              </td>
+              <td>Icon</td>
+              <td>
+                <Code>color-grey-700</Code>
+              </td>
+              <td>#666666</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>disabledIconColor</Code>
+              </td>
+              <td>Icon:disabled</td>
+              <td>
+                <Code>color-grey-500</Code>
+              </td>
+              <td>#999999</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>focusOutline</Code>
+              </td>
+              <td>Tab item outline</td>
+              <td>
+                <Code>color-purple-700</Code>
+              </td>
+              <td>#5f249f</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>selectedUnderlineColor</Code>
+              </td>
+              <td>Tab item border botton</td>
+              <td>
+                <Code>color-purple-700</Code>
+              </td>
+              <td>#5f249f</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>dividerColor</Code>
+              </td>
+              <td>Separator</td>
+              <td>
+                <Code>color-grey-400</Code>
+              </td>
+              <td>#bfbfbf</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>badgeBackgroundColor</Code>
+              </td>
+              <td>Badge container</td>
+              <td>
+                <Code>color-red-700</Code>
+              </td>
+              <td>#d0011b</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>badgeFontColor</Code>
+              </td>
+              <td>Label</td>
+              <td>
+                <Code>color-white</Code>
+              </td>
+              <td>#ffffff</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Accessibility</HeadingLink>
+        <DxcText as="p">
+          Each tab must have a unique title that clearly describes tab panel
+          content. This is particularly helpful for users of assistive
+          technologies so they have the necessary information to efficiently
+          navigate the content.
+        </DxcText>
+        <DxcText as="p">
+          Content authors need to ensure the content that is added to the tab
+          panel is accessible. For example, if you add an image to the panel you
+          need to include alternative text to pass accessibility testing.
+        </DxcText>
+        <DxcText as="p">
+          <DxcLink
```

Comment:
It should have newWindow prop to open in a new tab.

Reply:
Done